### PR TITLE
Stellar: support for get address

### PR DESCRIPTION
**Pull request by tsusanka**
Updates https://github.com/trezor/trezor-core/issues/226 and https://github.com/trezor/trezor-mcu/issues/363

### CODE REVIEW

**Comment by prusnak** [comment]
LGTM utACK

File: trezorlib/client.py [hunk 1]
```diff
@@ -1102,6 +1102,10 @@ def self_test(self):
     def stellar_get_public_key(self, address_n):
         return self.call(proto.StellarGetPublicKey(address_n=address_n))
 
+    @expect(proto.StellarAddress)
```
Comment (matejcik) [inline trezorlib/client.py]: please add `@field('address')` above this line, and modify the tests accordingly.

(the equivalent should be done for stellar_get_public_key - if you have the time, please add that too :) )

**Comment by matejcik** [comment]
generally LGTM (apart from the `@field` thing), but there should be a corresponding trezorctl command.
care to add it?
(or i can do it, if you don't mind waiting until (probably) tomorrow)

**Comment by tsusanka** [comment]
I've fixed the `@field` annotation - also for get_public_key. I can add the trezorctl command, but also tomorrow.

**Comment by prusnak** [comment]
Added the trezorctl command as I needed that anyway for testing

**Comment by matejcik** [comment]
perfect, thanks. :+1:

**Pull request merged**